### PR TITLE
#5289 [Manager] Use all uppercase for buttons and tabs text

### PR DESCRIPTION
**Pull request by saivirtue**
### Checklist
- [x] [Review Contribution Guidelines](https://oharastream.github.io/en/docs/master/contributing/)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #5289)
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

**Comment by saivirtue** [comment]
retry manager-it

File: ohara-manager/client/cypress/e2e/devToolDialog.test.ts [hunk 1]
```diff
@@ -32,7 +32,7 @@ describe('DevToolDialog - Topics tab', () => {
     cy.findByTitle('Create a new workspace').should('be.visible');
 
     cy.findByTitle('Developer Tools').click();
-    cy.findByText(/^topics$/i).should('exist');
+    cy.findByText(/^TOPICS$/).should('exist');
```
Comment (konekoya) [inline ohara-manager/client/cypress/e2e/devToolDialog.test.ts:35]: Couldn't we just use "TOPICS" here?

Comment (saivirtue) [reply ohara-manager/client/cypress/e2e/devToolDialog.test.ts:35]: Should be...just want to reduce the code changes for this PR 😅 
Will update this PR later

**Comment by saivirtue** [comment]
retry manager-it

**Comment by saivirtue** [comment]
retry manager-it

**Comment by saivirtue** [comment]
retry manager-it

**Comment by saivirtue** [comment]
retry manager-e2e

**Pull request merged**